Road/notebooks/data_cleaning.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for i, row in data.iterrows():
</pre><p>Please use representative variable name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@zubaidasader Please check this conversation https://github.com/mawada-sweis/Road-Status/pull/14#discussion_r1145614611